packages/components/src/lib/controllers/app-controller.js

```diff
@@ -512,8 +449,8 @@ export default class AppController {
       isMousemoveForced: this.isMousemoveForced.bind(this),
       getMouseMoveOverride: this.getMouseMoveOverride.bind(this),
       clearMouseMoveOverride: this.clearMouseMoveOverride.bind(this),
-      featureOptionChange: this.featureOptionChange.bind(this),
-      resetAllOptions: this.resetAllOptions.bind(this),
+      applyOptions: this.applyOptions.bind(this),
```

Comment:
very nitpicky but if applyUIOption is singular maybe applyOptions should be too (applyOption) ?

Reply:
The difference here is that `applyOptions()` applies all options at once while `applyUIOption()` is responsible for changing a single UI option only. But maybe I should rename `applyOptions()` to something like `applyAllOptions()` to make this distinction clearer? What do you think?

Reply:
yes I guess so. It's not a big deal, but it might make the code a little clearer.

Reply:
I've renamed it